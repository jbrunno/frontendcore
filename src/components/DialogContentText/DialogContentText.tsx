import React from 'react';
import {
  DialogContentText as MuiDialogContentText,
  DialogContentTextProps as MuiDialogContentTextProps,
} from '@mui/material';

export type DialogContentTextProps = MuiDialogContentTextProps;

export function DialogContentText(props: DialogContentTextProps) {
  return <MuiDialogContentText {...props} />;
}

export default DialogContentText;
