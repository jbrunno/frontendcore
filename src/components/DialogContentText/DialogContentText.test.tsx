import React from 'react';
import { render } from '@test-utils/component-testing';
import { DialogContentText } from './DialogContentText';

test('renders the DialogContentText', () => {
  const { container } = render(<DialogContentText />);
  expect(container).toMatchSnapshot();
});
