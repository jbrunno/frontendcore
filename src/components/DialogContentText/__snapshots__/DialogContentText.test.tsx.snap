// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the DialogContentText 1`] = `
.emotion-0 {
  margin: 0;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5rem;
  letter-spacing: 0.0094rem;
  font-family: Lato,sans-serif;
  color: #3B3B4199;
}

<div>
  <p
    class="MuiTypography-root MuiDialogContentText-root MuiTypography-body1 emotion-0"
  />
</div>
`;
